CMakeLists.txt

```diff
@@ -125,6 +125,8 @@ rocm_setup_version(VERSION ${version})
 
 list(APPEND CMAKE_PREFIX_PATH ${CMAKE_INSTALL_PREFIX} ${CMAKE_INSTALL_PREFIX}/llvm ${CMAKE_INSTALL_PREFIX}/hip /opt/rocm /opt/rocm/llvm /opt/rocm/hip "$ENV{ROCM_PATH}" "$ENV{HIP_PATH}")
 
+option(CK_BUILD_HOST_LIB, "Only build the CK JIT Helper Library" OFF)
```

Comment:
Remove these changes. CK already has the `CK_USE_CODEGEN`. Furthermore, they also intend to use the codegen library as a standalone rather than integrated build for now, so there is no reason to support this scenario.